core/src/main/scala/chisel3/choice/package.scala

```diff
@@ -3,8 +3,13 @@
 package chisel3
 
 import chisel3.experimental.{BaseModule, SourceInfo}
+import chisel3.internal.Builder
 import chisel3.util.simpleClassName
 
+import scala.collection.mutable
+import scala.reflect.runtime.universe._
+import scala.reflect.runtime.{currentMirror => cm}
```

Comment:
`scala.reflect._` doesn't cross-compile with Scala 3, it needs to be put in `core/src/main/scala-2/chisel3/`